File: src/typescript/transaction/working-with-transaction/binary-view/from-binary.ts
import { createHiveChain } from '@hiveio/wax';

// Initialize hive chain interface
const chain = await createHiveChain();

// Deserialize transaction from binary form
const tx = chain.convertTransactionFromBinaryForm(
  '8059b32ca6018b9fb568010003677467036774670b68656c6c6f2d776f726c6498080000'
);

// Display our transaction - note: This will create a transaction in Hive API-format
console.log(tx);

File: src/rust/transaction/working_with_transaction/binary_view/from_binary.rs
use wax::prelude::*;

fn main() -> Result<(), Box<dyn std::error::Error>> {
    // Initialize hive chain interface
    let chain = create_hive_chain(None)?;

    // Deserialize transaction from binary form
    let tx = chain.convert_transaction_from_binary_form(
        "8059b32ca6018b9fb568010003677467036774670b68656c6c6f2d776f726c6498080000",
    )?;

    // Display our transaction - note: This will create a transaction in Hive API-format
    println!("{tx:#}");

    Ok(())
}
